Orange/tests/test_distances.py

```diff
@@ -436,148 +442,155 @@ def test_spearmanr_distance_one_example(self):
-                                                 [ 0.25,  0.25,  0.25,  0.25,  0.25,  0.25,  0.  ,  0.25,  0.  ],
-                                                 [ 0.25,  0.  ,  0.25,  0.  ,  0.  ,  0.  ,  0.25,  0.  ,  0.25],
-                                                 [ 0.25,  0.25,  0.25,  0.25,  0.25,  0.25,  0.  ,  0.25,  0.  ]]))
+                                       np.array([[0., 0.25, 0., 0.25, 0.25, 0.25, 0.25, 0.25, 0.25],
+                                                 [0.25, 0., 0.25, 0., 0., 0., 0.25, 0., 0.25],
+                                                 [0., 0.25, 0., 0.25, 0.25, 0.25, 0.25, 0.25, 0.25],
+                                                 [0.25, 0., 0.25, 0., 0., 0., 0.25, 0., 0.25],
+                                                 [0.25, 0., 0.25, 0., 0., 0., 0.25, 0., 0.25],
+                                                 [0.25, 0., 0.25, 0., 0., 0., 0.25, 0., 0.25],
+                                                 [0.25, 0.25, 0.25, 0.25, 0.25, 0.25, 0., 0.25, 0.],
+                                                 [0.25, 0., 0.25, 0., 0., 0., 0.25, 0., 0.25],
+                                                 [0.25, 0.25, 0.25, 0.25, 0.25, 0.25, 0., 0.25, 0.]]))
         np.testing.assert_almost_equal(self.dist(self.breast[:3], self.breast[:4]),
-                                       np.array([[ 0.        ,  0.49166667,  0.075     ,  0.38333333],
-                                                 [ 0.49166667,  0.        ,  0.38333333,  0.46666667],
-                                                 [ 0.075     ,  0.38333333,  0.        ,  0.36666667]]))
+                                       np.array([[0., 0.49166667, 0.075, 0.38333333],
+                                                 [0.49166667, 0., 0.38333333, 0.46666667],
+                                                 [0.075, 0.38333333, 0., 0.36666667]]))
         np.testing.assert_almost_equal(self.dist(self.breast[3], self.breast[:4]),
-                                       np.array([[ 0.3833333,  0.4666667,  0.3666667,  0.       ]]))
+                                       np.array([[0.3833333, 0.4666667, 0.3666667, 0.]]))
         np.testing.assert_almost_equal(self.dist(self.breast[:4], self.breast[3]),
-                                       np.array([[ 0.3833333],
-                                                 [ 0.4666667],
-                                                 [ 0.3666667],
-                                                 [ 0.       ]]))
+                                       np.array([[0.3833333],
+                                                 [0.4666667],
+                                                 [0.3666667],
+                                                 [0.]]))
         np.testing.assert_almost_equal(self.dist(self.breast[:3], self.breast[:3], axis=0),
-                                       np.array([[ 0.  ,  0.25,  0.  ,  0.25,  0.25,  0.25,  0.25,  0.25,  0.25],
```

Comment:
If matrices contain values that you need to understand, I agree. But nice formatting can also be achieved by using a fixed number of decimals (0.00 instead of 0.)

+1 for constants in a separate module, but not as a part of this PR. Distance tests need a more thorough cleanup anyway.